doc/distrib/NodeHelpFiles/_review/Revit.Elements.Room.CoreBoundary.md

```diff
@@ -0,0 +1,10 @@
+## In Depth
+`Room.CoreBoundary` returns a nested list representing the given room's core boundary. In the returned list, the first sublist represents the outermost curves, while subsequent lists represent loops within the room. Core boundaries occur on Revit rooms on the wall's section that is defined as core.
```

Comment:
Similar to the previous one, for clarity purposes suggest rephrasing the last sentence "Core boundaries occur..." to "The Core boundaries are located at the interior or exterior layer of the core that is closest to the room" Also thinking, a link to the documentation on different types of room location boundaries might be handy here:[](https://help.autodesk.com/view/RVT/2024/ENU/?guid=GUID-0BB62832-36DD-4E06-A9D4-EE98CE0FCF89)https://help.autodesk.com/view/RVT/2024/ENU/?guid=GUID-0BB62832-36DD-4E06-A9D4-EE98CE0FCF89